spec-main/api-app-spec.ts

```diff
@@ -673,26 +673,29 @@ describe('app module', () => {
   })
 
   describe('getPath("logs")', () => {
+    before(() => {
+      // this won't be deterministic except on CI
+      if (!isCI) return
+    })
```

Comment:
I think this function literally has no effect and that the tests run either way?

```js
before(function () {
  if (!isCI) this.skip()
})
```